code/model/DMSDocument.php

```diff
@@ -54,27 +55,22 @@ class DMSDocument extends DataObject implements DMSDocumentInterface
         'EditorGroups' => 'Group',
     );
 
+    private static $has_one = array(
+        "CreatedBy" => "Member",
+        "LastEditedBy" => "Member",
+    );
+
     private static $display_fields = array(
         'ID' => 'ID',
         'Title' => 'Title',
         'FilenameWithoutID' => 'Filename',
-        'LastChanged' => 'LastChanged'
+        'LastEhanged' => 'LastEdited'
     );
 
     private static $singular_name = 'Document';
 
     private static $plural_name = 'Documents';
 
-    private static $searchable_fields = array(
-        'ID' => array(
-            'filter' => 'ExactMatchFilter',
-            'field' => 'NumericField'
-        ),
-        'Title',
-        'Filename',
-        'LastChanged'
-    );
-
```

Comment:
Can you add any configuration changes you've made (including the new ones for the query builder) to an appropriate area of the documentation?